src/hooks/useSetupEngineManager.ts

```diff
@@ -93,7 +93,12 @@ export function useSetupEngineManager(
     engineCommandManager.settings = settings
 
     const handleResize = deferExecution(() => {
-      engineCommandManager.handleResize(engineCommandManager.streamDimensions)
+      engineCommandManager.handleResize(
+        getDimensions(
+          streamRef?.current?.offsetWidth ?? 0,
+          streamRef?.current?.offsetHeight ?? 0
+        )
+      )
```

Comment:
this is the fix, streamDimensions are just the initial stream dimensions, need to be updated on resize, not passed in as the resize dimensions.